.github/workflows/node.js.yml

```diff
@@ -24,4 +24,4 @@ jobs:
       working-directory: packages/react-components
     - run: yarn workspace @department-of-veterans-affairs/react-components run build
     - run: yarn workspace @department-of-veterans-affairs/web-components run build
-    - run: yarn workspaces foreach run test
+    - run: yarn workspaces foreach -v run test
```

Comment:
This `verbose` flag puts the workspace package name in front of each line of output so that we can have an easier time finding what's failing.